src/mobile/src/ui/components/TransferConfirmationModal.js

```diff
@@ -152,7 +152,7 @@ class TransferConfirmationModal extends Component {
                             ? message.length > 0
                                 ? t('sendingAMessage').toUpperCase()
                                 : t('sendingAnEmptyMessage').toUpperCase()
-                            : t('fromAccount', { selectedAccountName }).toUpperCase()}
+                            : t('fromAccount', { selectedAccountName: `${selectedAccountName.substring(0, 18)}...` }).toUpperCase()}
```

Comment:
We don't need to shorten account names that are under 18 characters

```suggestion
                            : selectedAccountName.length >= 18
                            ? t('fromAccount', {
                                  selectedAccountName: `${selectedAccountName.substring(0, 18)}...`,
                              }).toUpperCase()
                            : t('fromAccount', { selectedAccountName }).toUpperCase()}
```